README.md

```diff
@@ -15,12 +15,12 @@ There are basic rules of conduct that should be followed at all times by everyon
 - Do not use ALL CAPS when creating an issue report.
 - Do not repeatedly update an open issue remarking that the issue persists.
 
-Remember: Just because the issue you reported was reported here does not mean that it is an issue with CS:GO.  As well, should your issue not be resolved immediately, it does not mean that a resolution is not being researched or tested.  Patience is always appreciated.
+Remember: Just because the issue you reported was reported here does not mean that it is an issue with CS2.  As well, should your issue not be resolved immediately, it does not mean that a resolution is not being researched or tested.  Patience is always appreciated.
 
 Reporting Issues
 ----------------
 
-If you encounter a bug while using CS:GO, first search the [issue list](https://github.com/ValveSoftware/csgo-osx-linux/issues?q=is%3Aissue) to see if it has already been reported. Include closed issues in your search. If your issue has been reported, please upvote the issue by clicking the "Add Reaction" button (smiley face with a plus sign) on the root post and adding a thumbs up. Voting helps us determine which issues are important to users without cluttering the bug database. Duplicate issue reports may be closed without comment.
+If you encounter a bug while using CS2, first search the [issue list](https://github.com/ValveSoftware/csgo-osx-linux/issues?q=is%3Aissue) to see if it has already been reported. Include closed issues in your search. If your issue has been reported, please upvote the issue by clicking the "Add Reaction" button (smiley face with a plus sign) on the root post and adding a thumbs up. Voting helps us determine which issues are important to users without cluttering the bug database. Duplicate issue reports may be closed without comment.
```

Comment:
This line should also be updated when the repository name is also updated (even though it would redirect anyway).